### PR TITLE
chore: gh actions run install only if no cache

### PR DESCRIPTION
**Pull request by richbachman**
- [x] Changed PR checks and VRT check to only install if there is no yarn cache
- [x] Changed package size check to only build JS
- [x] Added `yarn build:js` to core scripts
- [x] Remove unbarrelled imports from docs site

**TODO: After merging, change the build scripts in Vercel to:**
- Paste: `yarn build:js && yarn build:tokens && yarn build:website`
- Theme designer: `yarn build:js && yarn build:tokens && yarn build:theme-designer`

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -23,8 +23,8 @@
     "watch:icons": "yarn workspace @twilio-paste/icons watch",
     "prebuild": "node ./tools/build/pre-build.js && yarn packages:check",
     "build": "lerna run build --ignore @twilio-paste/website --ignore @twilio-paste/theme-designer --ignore @twilio-paste/nextjs-template",
-    "build:js": "yarn prebuild && lerna run build:js --since HEAD --ignore @twilio-paste/website --ignore @twilio-paste/theme-designer",
-    "build:dev": "yarn prebuild && lerna run build:dev --since HEAD --ignore @twilio-paste/website",
+    "build:js": "yarn prebuild && lerna run build:js --ignore @twilio-paste/website --ignore @twilio-paste/theme-designer",
+    "build:dev": "yarn prebuild && lerna run build:dev --ignore @twilio-paste/website",
```
Comment (richbachman) [inline package.json:27]: Does anyone use this one?

Comment (TheSisb) [reply package.json:27]: Nah, let's remove it. I removed it everywhere except missed this one.

**Comment by TheSisb** [comment]
image
Nice!

File: .github/workflows/vrt_pull_request.yml [hunk 2]
```diff
@@ -34,13 +35,15 @@ jobs:
...
         uses: actions/cache@v2
+        id: node_modules_cache_id
         with:
           path: |
             node_modules
             */*/node_modules
           key: ${{ runner.os }}-${{ hashFiles('**/yarn.lock') }}
 
       - name: Install Dependencies
+        if: steps.yarn_cache_id.outputs.cache-hit != 'true' || steps.node_modules_cache_id.outputs.cache-hit != 'true'
```
Comment (TheSisb) [inline .github/workflows/vrt_pull_request.yml:46]: 🔥

**Review by TheSisb — APPROVED** [review]
Solid improvements. 🚢

**Pull request merged**